VimToDatasmith/CVimToDatasmith.h

```diff
@@ -0,0 +1,204 @@
+//
+//  CVimToDatasmith.h
+//  CVimToDatasmith
+//
+//  Created by Richard Young on 2021-05-17.
+//
```

Comment:
For headers throughout let's just use

```suggestion
// Copyright (c) 2021 VIM
// Licensed under the MIT License 1.0
```